practice/pyramid/site.css

```diff
@@ -0,0 +1,38 @@
+
+.row {
+  display: flex;
+  border: 1px solid black;
+  margin: 20px;
+  height: 100px;
+  justify-content: space-between;
+}
+
+.first-row-item {
+  flex: 1;
+  margin: 10px;
+  border: 1px solid black;
+}
+
+.second-row-item1 {
+  flex: 4;
+  margin: 10px;
+  border: 1px solid black;
+}
+
+.second-row-item2 {
+  flex: 1;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-left: 0;
+  margin-right: 10px;
+  border: 1px solid black;
+}
+
+.third-row-item {
+  flex: 1;
+  margin: 10px;
+  flex-basis: 90px;
+  flex-grow: 0;
+  flex-shrink: 0;
```

Comment:
That's because specifying both `flex: 1` and `width: 90px` are in direct conflict. Both describe the width of the element. You should only set one of the two (and then setting `width` means all of the other `flex-*` properties are not necessary). My tests show that produces the correct result.